app/main/controllers/activity/RTMediaActivity.php

```diff
@@ -53,71 +58,72 @@ function create_activity_html( $type = 'activity' ) {
 			$limit_activity_feed = 0;
 		}
 
-		$mediaObj      = new RTMediaModel();
-		$media_details = $mediaObj->get( array( 'id' => $this->media ) );
+		$rtmedia_model = new RTMediaModel();
+		$media_details = $rtmedia_model->get( array( 'id' => $this->media ) );
 
 		if ( intval( $limit_activity_feed ) > 0 ) {
 			$media_details = array_slice( $media_details, 0, $limit_activity_feed, true );
 		}
-		$rtmedia_activity_ul_class = apply_filters( 'rtmedia_'.$type.'_ul_class', 'rtm-activity-media-list' );
-		$li_content                = '';
-		$count                     = 0;
+		$rtmedia_activity_ul_class = apply_filters( 'rtmedia_' . $type . '_ul_class', 'rtm-activity-media-list' );
+
+		$media_content = '';
+		$count         = 0;
 		foreach ( $media_details as $media ) {
-			$li_content .= '<li class="rtmedia-list-item media-type-' . esc_attr( $media->media_type ) . '">';
-			if ( 'photo' === $media->media_type || 'document' === $media->media_type || 'other' === $media->media_type  ) {
-				$li_content .= '<a href ="' . esc_url( get_rtmedia_permalink( $media->id ) ) . '">';
+			$media_content .= '<li class="rtmedia-list-item media-type-' . esc_attr( $media->media_type ) . '">';
+			if ( 'photo' === $media->media_type || 'document' === $media->media_type || 'other' === $media->media_type ) {
+				$media_content .= '<a href ="' . esc_url( get_rtmedia_permalink( $media->id ) ) . '">';
 			}
-			$li_content .= '<div class="rtmedia-item-thumbnail">';
+			$media_content .= '<div class="rtmedia-item-thumbnail">';
 
-			$li_content .= $this->media( $media );
+			$media_content .= $this->media( $media );
 
-			$li_content .= '</div>';
+			$media_content .= '</div>';
```

Comment:
we can use this in one line.

$media_content .= sprintf( '<div class="rtmedia-item-thumbnail">%s</div>', esc_html(  $this->media( $media ) ) );

check if require escaping in media.